extensions/itchio.js

```diff
@@ -278,20 +318,25 @@
       return data.id ? true : false;
     }
     returnGameRewards({ index, rewards }) {
-      try {
-        return data.rewards[index][rewards];
-      } catch (e) {
-        return err();
-      }
+      return data.rewards?.[index]?.[rewards] || err();
     }
     rewardsLenght() {
-      return data.rewards ? data.rewards.lenght : 0;
+      return data.rewards?.lenght || 0;
     }
     rewardsBool() {
       return data.rewards ? true : false;
     }
+    returnGameSubProducts({ subProducts, index }) {
+      data.sub_products?.[index]?.[subProducts] || err();
+    }
+    subProductsLenght() {
+      return data.sub_products?.lenght || 0;
```

Comment:
and here

Reply:
all fixed except one opcode changing which will break compatibility